dubbo-registry/dubbo-registry-redis/src/main/java/org/apache/dubbo/registry/redis/RedisRegistry.java

```diff
@@ -428,7 +428,7 @@ private void doNotify(Jedis jedis, Collection<String> keys, URL url, Collection<
             }
             List<URL> urls = new ArrayList<>();
             Map<String, String> values = jedis.hgetAll(key);
-            if (CollectionUtils.isEmptyMap(values)) {
+            if (!CollectionUtils.isEmptyMap(values)) {

```

Comment:
Thanks for this PR. Could you add a UT for this?

Reply:
Is it necessary? This is my first PR，I don't know much about。

Reply:
@boomblog no worries I will add that on behalf of you 😃 . As a practice we follow to add UT if there is conditional changes. Do us a favour create an issue with information like UT and provide a reference of this PR.

Thanks for raising this PR and the contribution. Welcome.